pkg/pmem-csi-operator/controller/deployment/controller_driver.go

```diff
@@ -1046,6 +1196,69 @@ func (d *PmemCSIDriver) getObjectMeta(name string, isClusterResource bool) metav
 	return meta
 }
 
+func (d *PmemCSIDriver) objectForGVK(gvk schema.GroupVersionKind, name string) apiruntime.Object {
```

Comment:
This entire function also disappears when not hand-coding `getDeploymentOwnedObjects`.